Description: remove interactive choice at launch
 it is no longer useful nowadays
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Last-Update: 2020-04-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/cool_gfx/gfx_sdl.c
+++ b/cool_gfx/gfx_sdl.c
@@ -34,61 +34,22 @@
 
 int sdl_init(int x,int y,int special)
 {
-    int mode;
+	doubled=0;
 
-    doubled=0;
+	if(SDL_Init(SDL_INIT_VIDEO)<0)
+			return(GFX_ERR_MODE);
 
-    if(special&GFX_NOASK)
-    {
-        mode=1;
-    }
-    else    /*  Prompt the user to select mode  */
-    {
-        printf("Internal window size: %dx%d\n",x,y);
-        puts("1 - Normal pixels");
-        puts("2 - Double pixels");
-        puts("3 - Normal pixels fullscreen (if possible)");
-        puts("4 - Double pixels fullscreen (if possible)");
-        scanf("%d",&mode);
-    }
+	x_size=x;
+	y_size=y;
 
-    if(SDL_Init(SDL_INIT_VIDEO)<0)
-                    return(GFX_ERR_MODE);
-
-    x_size=x;
-    y_size=y;
-
-
-    switch(mode)
-    {
-        case 1: sdlWindow = SDL_CreateWindow("One Day Miracle by Fit",SDL_WINDOWPOS_CENTERED,SDL_WINDOWPOS_CENTERED,x,y,SDL_WINDOW_OPENGL);
-                if(!sdlWindow)
-                    return(GFX_ERR_MODE);
-	        screen = SDL_CreateRGBSurfaceWithFormat(0, x, y, 32,SDL_PIXELFORMAT_RGB888); //buffer video
-                break;
-        case 2: sdlWindow = SDL_CreateWindow("One Day Miracle by Fit",SDL_WINDOWPOS_CENTERED,SDL_WINDOWPOS_CENTERED,x*2,y*2,SDL_WINDOW_OPENGL);
-                if(!sdlWindow)
-                    return(GFX_ERR_MODE);
-	        screen = SDL_CreateRGBSurfaceWithFormat(0, x*2, y*2, 32,SDL_PIXELFORMAT_RGB888); //buffer video
-                doubled=1;
-                break;
-        case 3: sdlWindow = SDL_CreateWindow("One Day Miracle by Fit",SDL_WINDOWPOS_CENTERED,SDL_WINDOWPOS_CENTERED,x,y,SDL_WINDOW_OPENGL | SDL_WINDOW_FULLSCREEN);
-                if(!sdlWindow)
-                    return(GFX_ERR_MODE);
-	        screen = SDL_CreateRGBSurfaceWithFormat(0, x, y, 32,SDL_PIXELFORMAT_RGB888); //buffer video
-                break;
-        case 4: sdlWindow = SDL_CreateWindow("One Day Miracle by Fit",SDL_WINDOWPOS_CENTERED,SDL_WINDOWPOS_CENTERED,x*2,y*2,SDL_WINDOW_OPENGL | SDL_WINDOW_FULLSCREEN);
-                if(!sdlWindow)
-                    return(GFX_ERR_MODE);
-	        screen = SDL_CreateRGBSurfaceWithFormat(0, x*2, y*2, 32,SDL_PIXELFORMAT_RGB888); //buffer video
-                doubled=1;
-                break;
-        default: return(GFX_ERR_MODE);
-    }
+	sdlWindow = SDL_CreateWindow("One Day Miracle by Fit",SDL_WINDOWPOS_CENTERED,SDL_WINDOWPOS_CENTERED,x*2,y*2,0);
+	if(!sdlWindow)
+	    return(GFX_ERR_MODE);
+	screen = SDL_CreateRGBSurfaceWithFormat(0, x*2, y*2, 32,SDL_PIXELFORMAT_RGB888); //video buffer
+	doubled=1;
 
     sdlRenderer = SDL_CreateRenderer(sdlWindow, -1, 0);
     sdlTexture = SDL_CreateTextureFromSurface(sdlRenderer,screen); // texture with format from video buffer
-
     SDL_ShowCursor(SDL_DISABLE);
     return(GFX_OK);
 }
--- a/cool_mzx/cool_mzx.c
+++ b/cool_mzx/cool_mzx.c
@@ -149,31 +149,8 @@
 /*      Alustaa musiikkisysteemin, kyselee laitteet yms.        */
 int mzx_init(void)
 {
-	char	pag[100];
-
-	puts("Select your sound device:");
-	puts("0 - None");
-	puts("1 - 8-bit 22kHz");
-	puts("2 - 16-bit 44kHz");
-	scanf("%s",pag);
-		
-	switch(pag[0])
-	{
-		case '0': laite=0;
-			  sd_ns_init(0,0,0);
-			  break;
-		case '1':
-			laite=1;
-			if(sd_dsp16_init(22050,0,0)==SD_ERR)
-			return(0); 
-			  break;
-		case '2': laite=2;
-			  if(sd_dsp16_init(44100,0,0)==SD_ERR)
-				return(0); 
-			  break;
-		default: return(0);
-	}
-
+	laite=2;
+	if(sd_dsp16_init(44100,0,0)==SD_ERR)
         callbackcount = 0;
         sd_init();
 	sd_setmainvolume(64); 
